sdk/src/beta9/cli/config.py

```diff
@@ -146,7 +146,8 @@ def create_context(config_path: Path, **kwargs):
     contexts[name] = context
     save_config(contexts=contexts, path=config_path)
 
-    terminal.success("Added new context 🎉!")
+    rel_path = str(config_path).replace(str(Path.home()), "~")
```

Comment:
I don't think `~` will work for users on windows machines

Reply:
Updated this to just use the path.